src/editor/mod.rs

```diff
@@ -174,6 +203,7 @@ impl Editor {
                     rows,
                     columns,
                 } => {
+                    tracy_zone!("EdtitorScroll");
```

Comment:
Minor typo

Reply:
This is fixed both here, and in the profiling PR, which actually adds the tracy zone